backend/app.py

```diff
@@ -384,6 +384,11 @@ def _cset_members_items(codeset_ids: Union[str, None] = Query(default=''), ) ->
     codeset_ids: List[int] = parse_codeset_ids(codeset_ids)
     return get_cset_members_items(codeset_ids)
 
+@APP.get("/db-refresh")
+def db_refresh_route():
```

Comment:
Looks correct. I wonder if the only reason it didn't print is because when a change is made like this to the backend code, the server needs to be restarted! IDK if Siggie had a chance to go over that. It's not like the frontend unfortunately, which refreshes every time it detects a file save w/ changes.

Reply:
Hi Joe, I actually did try restarting the backend but it didn't work

Reply:
Ahh, ok, I see what happened. My bad @forbeskat , I led you down a wrong path. I saw that the `UploadCSV.js` used a function called `axiosPut` which I thought was imported from `axios`, but it actually is a function in TermHub's `State.js`. 

I was going to import `axiosPut` from `State.js`, but to make things simpler I used `axiosGet` instead. We may change this to a `POST` in the future if we change it so that only certain users can do the DB refresh, but for now a `GET` is fine.

Anyway after importing `axiosGet` from `State.js` and using that instead, it's working now!